ecl/eclcc/eclcc.cpp

```diff
@@ -175,11 +175,12 @@ static bool getHomeFolder(StringBuffer & homepath)
     return true;
 }
 
-struct EclCompileInstance
+class EclCC;
+struct EclCompileInstance : public CInterfaceOf<ICodegenContextCallback>
```

Comment:
Is this change related to the changes proposed in the jira ticket? is this just code refactoring?

Reply:
Related to the ticket - the information to implement it is known by EclCompileInstance, but not by EclCC class, so all requests need to go via that class instead of direct to eclcc.